roster/src/Roster.Core/Domain/Rank.cs

```diff
@@ -4,8 +4,7 @@ namespace Roster.Core.Domain
 {
     public class Rank : AggregateRoot
     {
-        [SuppressMessage("Style", "IDE0044:Add readonly modifier", Justification = "Autogenerated by EF Core")]
-        private int _id;
+        private int _id = -1;
```

Comment:
Is this a dirty workaround to make the DB compatible with new EF version?

Reply:
Nah, it's to avoid a warning without suppressing it.